extensions/arc/src/ui/components/initializingComponent.ts

```diff
@@ -37,7 +37,11 @@ export abstract class InitializingComponent {
 		if (!this._initialized) {
 			this.onInitializedPromise.promise.then(() => action()).catch(error => console.error(`Unexpected error running onInitialized action: ${error}`));
 		} else {
-			action();
+			try {
+				action();
+			} catch (error) {
+				console.error(`Unexpected error running onInitialized action: ${error}`);
```

Comment:
Why are you doing this? We generally shouldn't be doing this - the only reason I have it on the promise is because there's nothing we could do anyways at that point for the call.

Reply:
ok, will remove.

Reply:
removed.